src/runtime/pay-client.js

```diff
@@ -342,13 +109,33 @@ export class PayClientBindingPayjs {
     );
   }
 
-  /** @override */
+  /**
+   * @param {!../utils/preconnect.Preconnect} pre
+   */
+  preconnect(pre) {
+    pre.prefetch(payUrl());
+    pre.prefetch(
+      'https://payments.google.com/payments/v4/js/integrator.js?ss=md'
+    );
+    pre.prefetch('https://clients2.google.com/gr/gr_full_2.0.6.js');
+    pre.preconnect('https://www.gstatic.com/');
+    pre.preconnect('https://fonts.googleapis.com/');
+    pre.preconnect('https://www.google.com/');
+  }
+
+  /**
+   * @return {string}
+   */
   getType() {
+    // TODO(alin04): remove once all references removed.
     return 'PAYJS';
   }
 
-  /** @override */
-  start(paymentRequest, options) {
+  /**
+   * @param {!Object} paymentRequest
```

Comment:
Can we make payment request a specific kind of object?

Reply:
Done